File: NOVEMBER-2025/09-11-2025/p2169.kt
class Solution {
    fun countOperations(num1: Int, num2: Int): Int {
        var count = 0
        var max = maxOf(num1, num2)
        var min = minOf(num1, num2)
        while (min != 0 && max != 0) {
            count += max / min
            val t = min
            min = max % min
            max = t
        }
        return count
    }
}

File: NOVEMBER-2025/09-11-2025/p2169.swift
class Solution {
    func countOperations(_ num1: Int, _ num2: Int) -> Int {
        var first = num1
        var second = num2
        var count = 0 
        while first != 0 && second != 0 {
            if first == second {
                count += 1
                return count
            } else if first > second {
                count += 1
                first = first - second
            } else {
                count += 1
                second = second - first
            }
        }
        return count
    }
}